slides/workshopslides_gitDevOps.html

```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
 
 <section id="what-is-git" class="title-slide slide level1 center">
 <h1>What is git?</h1>
-<ul>
-<li>Git is a piece of software designed for version control, i.e.
+<div class="fragment fade-up">
+<p>Git is a piece of software designed for version control, i.e.</p>
+</div>
+<div class="fragment fade-up">
 <ul>
 <li>Tracking changes;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Undoing changes;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Maintaining parallel variants of code.</li>
-</ul></li>
-<li>Git works with repositories:
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
+<li>Git works with repositories:</li>
+</ul>
+</div>
+<div class="fragment fade-up">
 <ul>
 <li>A repository (or repo) is simply a folder containing all the version controlled files of a given project.</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>It can have multiple sub-folders that will can also be tracked.</li>
-</ul></li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Git records all the version control information within a hidden folder called .git</li>
 </ul>
+</div>
 </section>
 
 <section id="and-github-and-dev-ops" class="title-slide slide level1 center">
 <h1>And GitHub and Dev Ops…</h1>
+<div class="fragment fade-up">
 <p>GitHub and Dev Ops provide a space to store and interact with a repository.</p>
```

Comment:
I'm pretty sure you say it out loud but some people in the workshops still get confused about git as the overarching "entity" vs GitHub/DevOps being a way to interact with that entity. People think git == GitHub and then they panic that their code/data is going to go somewhere public if they use git.

Reply:
Good point. Have added this explicitly in the slides now. Or at least I have locally, will push changes up once I've got internet again!